Library/Formula/emp.rb

```diff
@@ -0,0 +1,40 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/emp"
+  url "https://github.com/remind101/emp/archive/v0.9.1.tar.gz"
+  sha256 "9619ffd25856988d92d39b31d9e7ed6edacbd85543ff6e60300954df7c16135d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    emppath = buildpath/"src/github.com/remind101/emp"
+    emppath.install Dir["{*,.git}"]
+
+    cd emppath do
+      system "go", "build", "-o", bin/"emp"
+    end
+  end
+
+  test do
+    require "webrick"
+
+    server = WEBrick::HTTPServer.new :Port => 8035
+    server.mount_proc "/apps/foo/releases" do |_req, res|
+      res.body = '[{"created_at":"2015-10-12T00:00:00.00000000-04:00","description":"my awesome release","id":"v1","user":{"id":"zab","email":"zab@waba.com"},"version":1}]'
+    end
+
+    Thread.new { server.start }
+
+    begin
+      ENV["EMPIRE_API_URL"]="http://127.0.0.1:8035"
+      assert_equal "v1  zab  Oct 12 00:00  my awesome release",
```

Comment:
I believe the `emp` tool prints the time out in whatever timezone you are in. What about doing an assert match with a regex like:

```
/v1  zab  Oct 1(1|2|3) \d\d:00  my awesome release/
```

That way it doesn't matter what timezone you are in.